[2024-06-27T15:40:16.580-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T15:40:16.602-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:06:30+00:00 [queued]>
[2024-06-27T15:40:16.611-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:06:30+00:00 [queued]>
[2024-06-27T15:40:16.612-0400] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-27T15:40:16.654-0400] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_kafka_producer> on 2024-06-27 00:06:30+00:00
[2024-06-27T15:40:16.671-0400] {standard_task_runner.py:63} INFO - Started process 186013 to run task
[2024-06-27T15:40:16.678-0400] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'api_cuevana3_movie', 'run_kafka_producer', 'scheduled__2024-06-27T00:06:30+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/movie_recommend.py', '--cfg-path', '/tmp/tmpatvrb3ty']
[2024-06-27T15:40:16.682-0400] {standard_task_runner.py:91} INFO - Job 112: Subtask run_kafka_producer
[2024-06-27T15:40:16.753-0400] {task_command.py:426} INFO - Running <TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:06:30+00:00 [running]> on host walter-VirtualBox
[2024-06-27T15:40:16.886-0400] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_cuevana3_movie' AIRFLOW_CTX_TASK_ID='run_kafka_producer' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:06:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:06:30+00:00'
[2024-06-27T15:40:16.887-0400] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T15:40:16.914-0400] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T15:40:16.920-0400] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/venv/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/walter/Documentos/rakuntentv/airflow/dags/movie_recommend.py", line 41, in json_serialization
    producer = Producer(producer_config)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
cimpl.KafkaException: KafkaError{code=_INVALID_ARG,val=-186,str="No such configuration property: "bootstrap_server""}
[2024-06-27T15:40:16.930-0400] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=api_cuevana3_movie, task_id=run_kafka_producer, run_id=scheduled__2024-06-27T00:06:30+00:00, execution_date=20240627T000630, start_date=20240627T194016, end_date=20240627T194016
[2024-06-27T15:40:17.001-0400] {standard_task_runner.py:110} ERROR - Failed to execute job 112 for task run_kafka_producer (KafkaError{code=_INVALID_ARG,val=-186,str="No such configuration property: "bootstrap_server""}; 186013)
[2024-06-27T15:40:17.019-0400] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-27T15:40:17.044-0400] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T15:40:17.046-0400] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-27T16:02:14.870-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T16:02:14.896-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:06:30+00:00 [queued]>
[2024-06-27T16:02:14.903-0400] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:06:30+00:00 [queued]>
[2024-06-27T16:02:14.903-0400] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-27T16:02:14.924-0400] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_kafka_producer> on 2024-06-27 00:06:30+00:00
[2024-06-27T16:02:14.955-0400] {standard_task_runner.py:63} INFO - Started process 188896 to run task
[2024-06-27T16:02:14.959-0400] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'api_cuevana3_movie', 'run_kafka_producer', 'scheduled__2024-06-27T00:06:30+00:00', '--job-id', '173', '--raw', '--subdir', 'DAGS_FOLDER/movie_recommend.py', '--cfg-path', '/tmp/tmpb9_6kbny']
[2024-06-27T16:02:14.961-0400] {standard_task_runner.py:91} INFO - Job 173: Subtask run_kafka_producer
[2024-06-27T16:02:15.094-0400] {task_command.py:426} INFO - Running <TaskInstance: api_cuevana3_movie.run_kafka_producer scheduled__2024-06-27T00:06:30+00:00 [running]> on host walter-VirtualBox
[2024-06-27T16:02:15.503-0400] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_cuevana3_movie' AIRFLOW_CTX_TASK_ID='run_kafka_producer' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:06:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:06:30+00:00'
[2024-06-27T16:02:15.511-0400] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T16:02:15.632-0400] {python.py:237} INFO - Done. Returned value was: None
[2024-06-27T16:02:15.634-0400] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T16:02:15.668-0400] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_cuevana3_movie, task_id=run_kafka_producer, run_id=scheduled__2024-06-27T00:06:30+00:00, execution_date=20240627T000630, start_date=20240627T200214, end_date=20240627T200215
[2024-06-27T16:02:15.740-0400] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-27T16:02:15.759-0400] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T16:02:15.761-0400] {local_task_job_runner.py:222} INFO - ::endgroup::
